endpoints.json

```diff
@@ -107,5 +107,10 @@
 				"article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
 			}
 		}
-  }
+	},
+	"DELETE /api/comments/:comment_id": {
+		"description": "removes the comment specified by comment_id",
+		"queries": [],
+		"exampleResponse": "no response body"
```

Comment:
we should add an example request body here, showing how the user should provide the comment_id